ingest/expression_files/expression_files.py

```diff
@@ -138,6 +158,66 @@ def check_unique_cells(cell_names: List, study_id, client):
             raise ValueError(error_string)
         return True
 
+    @staticmethod
+    def has_expression_file_info_doc(client, study_id):
+        COLLECTION_NAME = "study_files"
+        QUERY = {"study_id": study_id, "expression_file_info": {"$exists": True}}
+        query_results = list(client[COLLECTION_NAME].find(QUERY))
+        if query_results:
+            return True
+        return False
+
+    @staticmethod
+    def generate_query_filters(
+        query_results, field_values, query_mappings: Dict
+    ) -> List[Dict]:
+        """Creates query filters
+
+        :parameter
+            query_results (List[Dict]) : Results from any query
+            field_values (List) : Keys that are taken from query_results
+            query_mappings : Map query_results fields to model attributes
+        Example:
+               query_results = [{_id: 1234}, {food:abc}, {barbara: streisand}]
+               field_values = [id, food]
+               query_mappings = {id: study_file_id, barbara: lastname}
+
```

Comment:
I don't understand what happened to the 'food' value in this example

Reply:
This function was removed